examples/kubernetes/addons/etcd-operator/development/cilium-development.yaml

```diff
@@ -137,6 +151,8 @@ spec:
         - "--kvstore=etcd"
         - "--kvstore-opt=etcd.config=/var/lib/etcd-config/etcd.config"
         - "--disable-ipv4=$(DISABLE_IPV4)"
+        - "--fixed-identity-mapping=128=kv-store"
```

Comment:
Why are these being added back? Do we still need these fixed identities when using the v1.3 image? 46292fcee05a41691d2a7b80c2c6ea6be2c24644 removed these lines:

```
diff --git a/examples/kubernetes/addons/etcd-operator/development/cilium-development.yaml b/examples/kubernetes/addons/etcd-operator/development/cilium-development.yaml
index bf04d55aa..0b589c62a 100644
--- a/examples/kubernetes/addons/etcd-operator/development/cilium-development.yaml
+++ b/examples/kubernetes/addons/etcd-operator/development/cilium-development.yaml
@@ -137,8 +137,6 @@ spec:
         - "--kvstore=etcd"
         - "--kvstore-opt=etcd.config=/var/lib/etcd-config/etcd.config"
         - "--disable-ipv4=$(DISABLE_IPV4)"
-        - "--fixed-identity-mapping=128=kv-store"
-        - "--fixed-identity-mapping=129=kube-dns"
         ports:
         - name: prometheus
           containerPort: 9090
```

Reply:
@ianvernon To have less diff between this file and the `examples/addons/etcd-operator/cilium-deployment.yaml` The fixed identity labels are only picked if the user has the fixed identity label in the annotation.

Reply:
OK, makes sense.